test/e2e/csi_test.go

```diff
@@ -220,4 +220,322 @@ var _ = Describe("Nutanix flavor CSI", Label("capx-feature-test", "csi"), func()
 
 		By("PASSED!")
 	})
+
+	It("Create a cluster with Nutanix CSI 3.0 and use Nutanix Volumes to create PV", Label("csi3"), func() {
```

Comment:
If other agree, I would like to not block this PR on this additional test merging.

I saw a related comment https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/460#issuecomment-2251294103 which will increase the scope of this fix.

IMO it would be more valuable to get a beta release out and let this be tested in different scenarios with "real"  clusters, while the e2e test is being worked on.

Reply:
I think we can take the suggestion in https://github.com/nutanix-cloud-native/cluster-api-provider-nutanix/pull/460#issuecomment-2251294103 as a separate PR in the future but that shouldn't entail removing the already working E2E